<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterDisplay</class>
 <widget class="QToolBox" name="CharacterDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ToolBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/control.jpg);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="page">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>313</width>
     <height>144</height>
    </rect>
   </property>
   <attribute name="label">
    <string>EventHorizon</string>
   </attribute>
   <widget class="QLabel" name="lable_greeting">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>313</width>
     <height>144</height>
    </rect>
   </property>
   <attribute name="label">
    <string>EventHorizon</string>
   </attribute>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
